source/draw/gpu/GpuContext.ooc

```diff
@@ -35,18 +35,7 @@ ToRasterFuture: class extends Future<RasterImage> {
 
 GpuContext: abstract class extends DrawContext {
 	defaultMap ::= null as Map
-	_defaultFontGpu: GpuImage = null
-	defaultFontGpu: GpuImage { get {
-		if (this _defaultFontGpu == null)
-			this _defaultFontGpu = this createImage(this defaultFontRaster)
-		this _defaultFontGpu
-	}}
```

Comment:
Having the atlas here would be a better design but destructors don't allow it and having two layers of destructors would be confusing.